notebook_demo.ipynb

```diff
@@ -43,22 +43,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Set your OpenAI API key as an environment variable. A good way to do this is to create a `.env` file with a single line:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 5,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "!echo \"OPENAI_API_KEY=<your key here>\" >> .env"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
+    "Set your OpenAI API key as an environment variable. A good way to do this is to create a `.env` file with a single line.\n",
```

Comment:
I use `echo` in that it's easy to do in Colab. I am still not clear about how to create files in Colab. And `"!echo \"OPENAI_API_KEY=<your key here>\" >> .env"` may be the easiest way.

Reply:
Yeah, let's revisit this later. There are other options, such as uploading a file: https://stackoverflow.com/a/53296722